src/popup/components/Tokens.vue

```diff
@@ -123,9 +134,21 @@ export default {
     vertical-align: middle;
   }
 
+  &.rg {
+    --icon-size: 16px;
```

Comment:
This is the default size set in line 113. I think there is no need to set it twice.